.github/workflows/__shared-ci.yml

```diff
@@ -19,6 +19,10 @@ jobs:
     needs: linter
     uses: ./.github/workflows/__test-action-get-image-metadata.yml
 
+  test-action-action-get-image-name:
```

Comment:
Nitpick there is an extra action

Reply:
don't get what is the change you are requesting